openedx/core/djangoapps/user_api/views.py

```diff
@@ -608,6 +612,26 @@ def _add_city_field(self, form_desc, required=True):
             required=required
         )
 
+    def _add_state_field(self, form_desc, required=False):
```

Comment:
@rlucioni Yes it will be hidden on edx.org, I confirmed it locally by running microsite with this field set to "optional" in ```REGISTRATION_EXTRA_FIELDS``` and it was hidden on edX registration form on ```localhost:8000``` and was available on microsite form on ```mit.localhost:8000```.

**localhost:8000**
<img width="1097" alt="screen shot 2016-03-23 at 12 02 49 pm" src="https://cloud.githubusercontent.com/assets/2851134/13978014/67827d26-f0ef-11e5-91fb-8b0b7a87f24c.png">

**mit.localhost:8000**
<img width="1102" alt="screen shot 2016-03-23 at 12 03 21 pm" src="https://cloud.githubusercontent.com/assets/2851134/13978024/76524ab6-f0ef-11e5-9b1a-f1f08259f21d.png">